Source/Core/DolphinWX/ConfigMain.cpp

```diff
@@ -322,6 +322,7 @@ void CConfigMain::InitializeGUILists()
 void CConfigMain::InitializeGUIValues()
 {
 	const SCoreStartupParameter& startup_params = SConfig::GetInstance().m_LocalCoreStartupParameter;
+	bool NTSC = SConfig::GetInstance().m_LocalCoreStartupParameter.bNTSC;
```

Comment:
No, this is called when the preference window is opened.  You might have guessed that from the fact that this code is in Core/DolphinWX...